### PR TITLE
Gp 112 fe navbar for public pages

### PR DESCRIPTION
**Pull request by thuanvmse161224**

### CODE REVIEW

File: src/Layout/GuestLayout/Navbar/Navbar.tsx [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+
+import LoginButton from './LoginButton'
+import NavItem from './NavItem'
+
+import logo from '/assets/logo.svg'
+
+import * as styles from './styles'
+
+
+const navigationItems = [
```
Comment (danielNg91) [inline src/Layout/GuestLayout/Navbar/Navbar.tsx]: add type for this const

File: src/Layout/GuestLayout/Navbar/styles.ts [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+	marginLeft: '15px',
+	marginRight: '15px',
+};
+
+export const linkStyle: CSSProperties = {
+	textDecoration: 'none',
+	color: 'white',
+};
+
+export const textLogoStyle = {
```
Comment (danielNg91) [inline src/Layout/GuestLayout/Navbar/styles.ts]: add type, use Object.freeze for all const in this file

**Review by danielNg91 — CHANGES_REQUESTED** [review]

File: src/Layout/GuestLayout/Navbar/Navbar.tsx (hunk 1 above)
Comment (thuanvmse161224) [reply src/Layout/GuestLayout/Navbar/Navbar.tsx]: working on it

Comment (thuanvmse161224) [reply src/Layout/GuestLayout/Navbar/Navbar.tsx]: done

File: src/Layout/GuestLayout/Navbar/styles.ts (hunk 2 above)
Comment (thuanvmse161224) [reply src/Layout/GuestLayout/Navbar/styles.ts]: done

File: src/Layout/GuestLayout/Navbar/styles.ts [hunk 3]
```diff
@@ -0,0 +1,50 @@
+import { Theme } from '@mui/material/styles/createTheme';
+import { SxProps } from '@mui/system';
+import { CSSProperties } from 'react';
+
+
+export const navbarStyle: CSSProperties = Object.freeze({
```
Comment (danielNg91) [inline src/Layout/GuestLayout/Navbar/styles.ts]: it is generic function, use Object.freeze<CSSProperties> to get advantage of type suggestion

Comment (thuanvmse161224) [reply src/Layout/GuestLayout/Navbar/styles.ts]: done

**Review by danielNg91 — APPROVED** [review]

**Pull request merged**